### DISCUSSION TITLE
Play video in 3d scene

### DISCUSSION THREAD

**Discussion by r0zbeh** [google/model-viewer#4955] [Q&A] [answered]
as you can see in the picture , create 3d scene and now i need to put video with sound  in scene and play it by click !( video rotate or move when 3d is rotate or rotate)

please can you Tell me how many ways I can do this ؟(best?)

![Image](https://github.com/user-attachments/assets/da977a9e-8250-4d36-be64-2e624c97bd2e)

**Comment by elalish** [comment]
We have an example of that: https://modelviewer.dev/examples/scenegraph/#animatedTexturesExample

**Comment by r0zbeh** [comment]
video is play  but no sound !!!

**Reply by elalish** [reply, marked as answer]
Ah yes, it's muted by default. There is documentation [here](https://modelviewer.dev/examples/scenegraph/#pickMaterialExample), but it's not the clearest. You want:
```
const videoTexture = mv.createVideoTexture(url);
videoTexture.source.element.muted = false;
```
